src/main/java/view/GameLibraryView.java

```diff
@@ -115,6 +136,55 @@ else if (evt.getPropertyName().equals("build")) {
             final String[] availableGames = gameLibraryViewModel.getState().getAvailableGames();
             setGameSelection(availableGames);
         }
+        else if (evt.getPropertyName().equals("filter")) {
```

Comment:
I feel like this could be done directly in the filter button action listener. The gameTypes could be set upon initializing the game library in the gameLibraryController.

Right now the game filter controller is doing two things: setting the game type, as well as filtering the game.